src/util/LocalStorageWorker.ts

```diff
@@ -0,0 +1,32 @@
+/** ****************************************************************************
+ * Copyright 2020-2020 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************** */
+
+import { EventMessage } from '../models/EventMessage';
+
+class LocalStorageWorker {
+	getPersistedPinnedMessages() {
+		const stringPersistedPinnedMessages = localStorage.getItem('pinnedMessages');
+		return stringPersistedPinnedMessages ? JSON.parse(stringPersistedPinnedMessages) : [];
```

Comment:
json parsing should be handled inside of try-catch block. also it's return type is any now. we should validate if it's message